### ISSUE TITLE
Parse R data files for readability

### ISSUE THREAD

**Issue by larsvilhuber** [AEADataEditor/replication-template-development#3]

**Comment by larsvilhuber** [comment]
See comments in https://github.com/AEADataEditor/replication-template-development/commit/5df50a79a0105e5a24ee10534c8cff25a2797609

**Comment by larsvilhuber** [comment]
@aaf101 I made a few changes to streamline/self-contain the program. Also corrected the "pattern" it was picking up files it shouldn't - needed to quote the ".". 

It is now picking up the right files. I created a test file (`touch test.rds` creates an empty file) that would NOT read, and it generates the following error messages:
```
Error in readRDS(datafiles_list[[k]]) : error reading from connection
Writing out results: /mnt/local/raid5_1/vilhuber/Workspace-non-encrypted/git/AEA/replication-template-development/r-data-checks.txt
Error in write.table(df, file = txtfile, sep = ",", quote = FALSE, row.names = FALSE) : 
  unimplemented type 'list' in 'EncodeElement'
> datafiles_list
[1] "/mnt/local/raid5_1/vilhuber/Workspace-non-encrypted/git/AEA/replication-template-development/12345/test.rds"
> df
                                                                                                    File name
1 /mnt/local/raid5_1/vilhuber/Workspace-non-encrypted/git/AEA/replication-template-development/12345/test.rds
  Successfully read
1                No
```

The first error (`Error in readRDS(datafiles_list[[k]]) : error reading from connection`) is expected (the file is not readable), but might be misleading - might be  suppressed?

The second error (`Error in write.table(df,`) is not clear to me, and will require a bit of debugging.

**Comment by aaf101** [comment]
@larsvilhuber I have suppressed the potentially confusing error message, and have (hopefully) resolved the `write.table(df)` error.

**Comment by larsvilhuber** [comment]
Merged in using Pull request #4
